lib/adrian/version.rb

```diff
@@ -1,3 +1,3 @@
 module Adrian
-  VERSION = '2.0.1'
+  VERSION = '2.0.2'
```

Comment:
I _think_ the `rake bump:patch` should do this for you (after the PR is approved/merged, we run `rake bump:[BUMP_REVISION]` followed by `rake release` which will publish it to our private repo)